### PR TITLE
Rewrite popup window move

### PR DESCRIPTION
**Pull request by michielvermeir**
The original algorithm for moving popup windows displayed some quirky behavior (e.g. Multiplicom/mastr_reporter#2029) and contained some magic numbers that I didn't really understand, so I changed it to something a bit more mundane.

### CODE REVIEW

**Comment by MichaelVyverman** [comment]
I get PopupWindow.js:476 Uncaught ReferenceError: $handle is not defined
    at HTMLDivElement.<anonymous> (PopupWindow.js:476)
    at HTMLDivElement.dispatch (jquery.js:5)
    at HTMLDivElement.v.handle (jquery.js:5) on mouse up

File: Windows/PopupWindow.js [hunk 1]
```diff
@@ -445,32 +445,36 @@ define([
...
-
-                    }
-                );
+              window._$ElContainer
+                .find(".AXMPopupWindowHeader")
+                .on("mousedown", function(e) {
+                  var $handle = $(this)
+                    .css("cursor", "move");
+                
+                var $window = $handle.parent()
```
Comment (piotrpawlaczek) [inline Windows/PopupWindow.js]: indent

File: Windows/PopupWindow.js [hunk 2]
```diff
@@ -445,32 +445,36 @@ define([
...
+                        .on("mouseup", function() {
+                          $(this).removeClass("AXMDraggable");
+                          $handle.css("cursor", "");
+                        });
+                    });
+                  e.preventDefault();
+                })
+                .on("mouseup", function() {
+                  $(".AXMDraggable").removeClass("AXMDraggable");
+                  $handle.css("cursor", "");
```
Comment (piotrpawlaczek) [inline Windows/PopupWindow.js]: $handle var is not declared in the current scope

**Comment by MichaelVyverman** [comment]
Some final comments: please remove the opacity (quit slow in an old macbook) and restrict the move so that the popup-window cannot leave the page on the right and header cannot leave the bottom of the page (left and top sides works already).

**Pull request closed without merging**